### PR TITLE
Add renderer for rich text content

### PR DESCRIPTION
**Pull request by jack-coggin**
- Adds new views to render rich text as correct html
- Adds new project for e2e testing with cypress

### CODE REVIEW

File: src/Dfe.ContentSupport.Web/Controllers/HomeController.cs [hunk 1]
```diff
@@ -5,14 +5,13 @@
...
 namespace Dfe.ContentSupport.Web.Controllers;
 
-public class HomeController(IContentfulService contentfulService)
-    : Controller
+public class HomeController(IContentfulService contentfulService) : Controller
 {
     public async Task<IActionResult> Index(string slug)
     {
         if (string.IsNullOrEmpty(slug)) return RedirectToAction("error");
+        var resp = await contentfulService.GetContent(slug) as ContentSupportPage;
```
Comment (Iain-Stanger) [inline src/Dfe.ContentSupport.Web/Controllers/HomeController.cs:13]: You might want to stay away from unnecessary casting here - have the method signature return the object type you want unless it's intended to be generic (in say, some kind of overload)

Comment (Iain-Stanger) [reply src/Dfe.ContentSupport.Web/Controllers/HomeController.cs:13]: @jack-coggin

File: src/Dfe.ContentSupport.Web/Common/RichTextTags.cs
Comment (simonjfirth) [inline src/Dfe.ContentSupport.Web/Common/RichTextTags.cs:1]: Swap to enub

File: src/Dfe.ContentSupport.Web/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
+@model ContentSupportPage
 
-<div class="text-center">
-    <h1 class="display-4">Welcome</h1>
-    <p>Learn about <a href="https://learn.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
-</div>
\ No newline at end of file
+<partial name="_Hero" model="@Model.Heading" />
+
+<main class="govuk-main-wrapper" id="main-content" role="main">
```
Comment (simonjfirth) [inline src/Dfe.ContentSupport.Web/Views/Home/Index.cshtml:6]: Remove as <main> tag already exists

**Review by simonjfirth — APPROVED** [review]

**Pull request merged**